src/main/java/org/aion/wallet/connector/api/ApiBlockchainConnector.java

```
@@ -138,8 +145,14 @@ public AccountDTO addKeystoreUTCFile(byte[] file, String password) throws Valida
         }
     }
 
-    private byte[] getPrivateKeyFromUTCKeystoreFile(byte[] key, String password) {
-        return KeystoreFormat.fromKeystore(key, password).getPrivKeyBytes();
+    public AccountDTO addPrivateKey(byte[] raw, String password) throws ValidationException {
+        try {
+            // todo: get correct public key from ECKey
+            ECKey key = ECKeyFac.inst().fromPrivate(raw);
+            return null;
```

I'd rather `throw new UnsupportedOperationException("addPrivateKey");` than `return null;`